ramalama/model_server.py

```
@@ -0,0 +1,98 @@
+            if value := entry.get(field_name):
+                models.append(str(value))
+                break
+    return models
+
+
+def _fetch_json(url: str, api_key: Optional[str] = None) -> tuple[int, Any]:
+    headers: dict[str, str] = {}
+    if api_key:
+        headers["Authorization"] = f"Bearer {api_key}"
+
+    request = urllib_request.Request(url, headers=headers, method="GET")
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)